x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -150,6 +154,67 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
     });
 
+    describe('#Single host Flyout', () => {
+      before(async () => {
+        await Promise.all([
```

Comment:
**nit**: in this case `Promise.all` is not necessary since we want to run sequentially this setup actions.
Also, `Promise.all` expect an array of promises, so using `await` inside it to resolve them is not what is meant for.